demo/test/components/previews/button_preview.rb

```diff
@@ -9,35 +9,35 @@ class ButtonPreview < Lookbook::Preview
   # @display classes flex flex-wrap space-x-2
 
   def default
-    render(Flowbite::Button.new(label: "Default"))
+    render(Flowbite::Button.new) { "Default" }
```

Comment:
For consistency's sake, that is the way to go. I still find it a bit odd that you cannot pass the (label) content to the initializer, like in an `options` hash or similar.